README.md

```diff
@@ -35,16 +35,16 @@ After logging into the Azure Portal, follow these steps to register the sample a
     4. Click **Create** to create the app.
 3. After the app has been created successfully, select the app from the list of applications. It should be at the bottom of the list.
 4. Copy the **Application ID** for the app you registered and paste it into two places:
-    * In the Web.config file on the line: `<add key="ida:ClientId" value="[ClientId]" />`
-    * In the run.csx file on the line: `private const string idaClientId = "[ClientId]";`
+    * In the [`Web.config`](https://github.com/jisungk/onedrive-data-robot-azure-function/blob/jisungk-edits/OneDriveDataRobot/Web.config) file on the line: `<add key="ida:ClientId" value="[ClientId]" />`
```

Comment:
Can you make these links relative links so they don't break when this code is integrated into the original project?